packages/toc/src/generators/python/index.ts

```diff
@@ -63,6 +63,20 @@ function generate(editor: IDocumentWidget<FileEditor>): IHeading[] {
       });
     };
   }
+
+  /**
+   * Clean a line of code that declares either a class or function/method
+   * keeps text until first open parenthesis or first colon
+   * i.e. strips out arguments of functions and superclass specification of classes
+   *
+   * @private
+   * @param line - line of python code to clean
+   * @returns cleaned line of code
+   */
+  function clean_declaration_code_string(line: string): string {
```

Comment:
In general the code base tends to use camel case names, so we could update this function to:

```suggestion
  function cleanDeclarationCodeString(line: string): string {
```

Reply:
Looks like ESLint didn't catch it because the rule if off?

https://github.com/jupyterlab/jupyterlab/blob/36713c57e28f1d974e90ee819e64e790b1dbeff8/.eslintrc.js#L41